bundles/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/internal/common/PckGenerator.java

```diff
@@ -777,4 +797,28 @@ private static int timeToRampValue(int timeMSec) {
         }
         return ret;
     }
+
+    /**
+     * Converts duration in milliseconds to lcntimer value
+     * Source: https://www.symcon.de/forum/threads/38603-LCN-Relais-Kurzzeit-Timer-umrechnen
+     *
+     * @param ms time in milliseconds
+     * @return lcn timer value
+     */
+    private static Integer convertMsecToLCNTimer(double ms) {
+        Integer lcntimer = -1;
+        if (ms >= 0 && ms <= 240960) {
+            double a = ms / 1000 / 0.03;
+            double b = (a / 32.0) + 1.0;
+            double c = Math.log(b) / Math.log(2);
+            double mod = Math.floor(c);
+            double faktor = 32 * (Math.pow(2, mod) - 1);
+            double offset = (a - faktor) / Math.pow(2, mod);
+            lcntimer = (new Double(offset + (mod * 32))).intValue();
+        } else {
+            LOGGER.error("Timer not in [0,240960] ms");
```

Comment:
Bindings should only log to error if something severe happened, like the detection of a bug in the code. You could make this warn.

Reply:
@fwolter Thanks for your hints and suggestions. I have changed the log level to warn. Furthermore I corrected upper bound regarding duration in startRelayTimer to 240960 (was 240000).

Hopefully the formatting issues where fixed too.